.github/workflows/ci-release.yml

```diff
@@ -1,7 +1,7 @@
 name: Release
 
 on:
-  create:
+  push:
```

Comment:
For my learning, what does changing `create`->`push` fix?

Reply:
I don't think create was a valid entry at all, hence the workflow kept running and failing on all PRs (because we check for tag in the script).